### PR TITLE
Update curated_attachments.lua

### PR DESCRIPTION
**Pull request by acjoker8818**
These should be all the new vehicles

### CODE REVIEW

File: lib/constructor/curated_attachments.lua [hunk 1]
```diff
@@ -1566,7 +1594,11 @@ local curated_attachments <const> = {
                               hash = 3663644634,
                               name = "Penumbra FF",
                               type = "VEHICLE"
-                          }, {
+                          },{
+                              hash = 1076201208,
+                              name = "300R",
+                              type = "VEHICLE"
+                          } {
```
Comment (hexarobi) [inline lib/constructor/curated_attachments.lua:1601]: missing a comma here?

Comment (acjoker8818) [reply lib/constructor/curated_attachments.lua:1601]: I dont think so? that is what it said the display name was

Comment (hexarobi) [reply lib/constructor/curated_attachments.lua:1601]: I mean between the objects =) `} {` should be `}, {`

**Review by acjoker8818 — COMMENTED** [review]
Maybe fixed it?

**Comment by hexarobi** [comment]
> Maybe fixed it?

thanks! looks good =)

**Pull request merged**